docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -36,11 +35,11 @@ spec:
 +
 [NOTE]
 ====
-* In a non-java application, that is a link:https://github.com/redhat-appstudio/build-definitions/blob/main/task/buildah/0.1/buildah.yaml[Buildah] task, adding the parameter mentioned above automatically isolates the build from the network, restricting it to only build from dependencies listed in your Git repository. If you need to pull in dependencies from a package manager for one of the xref:how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc#supported-languages[supported languages], see xref:how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc[Prefetching the package manager dependencies for the Hermetic build].
+* In a non-Java application that is a link:https://github.com/redhat-appstudio/build-definitions/blob/main/task/buildah/0.1/buildah.yaml[Buildah] task, adding the '**hermetic**' and '**true**' parameters automatically isolates the build from the network, which restricts it to building only from dependencies listed in your Git repository. If you need to pull in dependencies from a package manager for one of the xref:how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc#supported-languages[supported languages], see xref:how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc[Prefetching the package manager dependencies for the Hermetic build].
```

Comment:
> adding the '**hermetic**' and '**true**' parameters

It's parameter name and value, not two parameters. Also, worth to mention that true is a string, not bool.

Reply:
how does it read without making that specification?

Reply:
I think, this comment duplicates [this one](https://github.com/redhat-appstudio/docs.appstudio.io/pull/201#discussion_r1452301148).

Reply:
sorry @mmorhun i meant adding the part about string not bool.

Reply:
If user adds the param as bool, the whole pipeline will be invalid (no build will be started).

Reply:
@mmorhun what if i don't add this clarification at all? Would that be ok? i worry that being quite so granular could be confusing. if you think it's important to include, then is something like this what you mean? 

"[NOTE]
====
* In a non-Java application that is a...[Buildah] task, setting the '**hermetic**' parameter to '**true**', as a string and not a boolean, automatically isolates the build from the network, which restricts it to building only from dependencies listed in your Git repository. If you need to pull in dependencies from a package manager for one of the...."

Reply:
Yes, going so granular could confuse. Instead, let's have just an example (or link to it).

Reply:
ok thanks @mmorhun. can you link me to an example?

Reply:
and that's YAML, correct, @mmorhun?

Reply:
Yes, it is.